modules/rtticketview.php

```diff
@@ -36,6 +36,8 @@
 
 $ticket = $LMS->GetTicketContents($id);
 
+$ticket['childtickets'] = $LMS->GetChildTickets($id);
```

Comment:
Jeśli to mocno nie obciąża SQL to można wrzucić w treść GetTicketContents(), ale oczywiście samodzielne GetChildTickets() może i tak się przydać.

Reply:
Myślałem nad tym. Ale jak to ma pluć tylko numerkami i id to nie ma sensu robić tego.

Reply:
Teoretycznie gdyby GetTicketContents miałą w jakimś parametrze co ma wypluwać jako wynik to faktycznie można byłoby używać jej.

Reply:
Z grubsza ma - $short - zmieniłem w kilku miejscach ostatnio na `true`.
A można z $short zrobić taki parametr, który będzie mapą bitową co ma być zwrócone
i jeśli będzie `false` to odwzoruje się na zwrot informacji taki jak dotychczas, czyli jakaś predefiniowana mapa bitowa już po nowemu. Podobnie dla `true`, ale jeśli poda się wartość całkowitą to wtedy już potraktuje dosłownie jako mapę bitową.

Reply:
Proponuję zostawić tak jak jest w tej chwili. Ta funkcja jeszcze się przyda w kilku miejscach.

Reply:
Ciekawe jest to co proponujesz ale to są tematy na jakiś grubszy #codecleanup sam pewnie wiesz ile kodu jest zduplikowanego jeśli się tego obawiasz...